vsutil.py

```diff
@@ -103,7 +103,7 @@ def is_image(filename: str) -> bool:
     """
     Returns true if a filename refers to an image.
     """
-    return bool(re.search(r'\.(png|jpe?g|bmp|tiff?)$', filename))
+    return mimetypes.types_map[os.path.splitext(filename)[-1]].startswith("image/")
```

Comment:
🤔 I think I forgot to add an import.

```python
import mimetypes
```

Reply:
Did that for you.

Reply:
Yeah, it also appears to cause issues with anything that isn't an image when trying to use it in my own func now. Evidently I did not test this enough. :feelsdumbman: Looking into it atm

Reply:
This should fix it. Since `types_map` is apparently a dict. So we use `dict.get(k, d)`

```suggestion
    return mimetypes.types_map.get(os.path.splitext(filename)[-1], "").startswith("image/")
```

Reply:
Tested it (alongside double-checking if it works as intended while I was it), and it works. Thanks a stack!